src/Uno.UI/UI/Xaml/UIElement.wasm.cs

```diff
@@ -438,7 +443,7 @@ public bool Dispatch(EventArgs eventArgs, string nativeEventPayload)
 			registration.Add(handler);
 		}
 
-		internal void UnregisterEventHandler(string eventName, Delegate handler)
+		public void UnregisterEventHandler(string eventName, Delegate handler)
```

Comment:
Not supposed to be public. Use the InternalsVisibleTo for now.

Reply:
It makes sense to make this public: required to implement external component and subscribe to native JS events.

Reply:
Next commit will move this to an extension class.